env/utils/amd_arch.py

```
@@ -9,13 +9,13 @@
     "gfx942": "AMD CDNA3: MI300X, MI300A",
 }
 
-AMD_ARCH_LIST = [k for k in AMD_ARCH_DESC.keys()]
+AMD_ARCH_LIST = [k for k in AMD_ARCH_DESC]
```

Ruff (0.16.0)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)